packages/crypto/src/validation/schemas.ts

```diff
@@ -56,6 +56,11 @@ export const schemas = {
         allOf: [{ type: "string", pattern: "^[a-zA-Z0-9_-]+$" }, { minLength: 1, maxLength: 40 }],
     },
 
+    uri: {
+        $id: "uri",
+        allOf: [{ type: "string" }, { format: "uri" }, { minLength: 4, maxLength: 50 }],
```

Comment:
Is the `{ type: "string" }` part needed if we already have the URI expectation which logically would have to be a string?

Reply:
The ajv documentation could be interpreted as that it is required. 

> The following formats are supported for **string validation** with "format" keyword:
Source: https://www.npmjs.com/package/ajv#formats

However, I tested it without it and that seemed to work fine also.

Reply:
Would you like me to remove it?

Reply:
We should probably also increase the `maxLength`. 50 is quite short as some people might have really long URLs, especially private chains or repositories hosted on custom domains.

Reply:
Good point. How about 80? 100 feels a bit excessive to me.

Reply:
Lets go with 80 for now, we can increase it later on if it doesn't seem sufficient.